polyfill/test/helpers/temporalHelpers.js

```diff
@@ -480,6 +553,40 @@ var TemporalHelpers = {
     assert.compareArray(actual, expected, "property getters not called");
   },
 
+  /*
+   * A custom calendar that returns an iterable instead of an array from its
+   * fields() method, otherwise identical to the ISO calendar.
+   */
+  calendarFieldsIterable() {
```

Comment:
Should we have a follow up test262 work item (not needed for this PR) to test that returning an iterator with additional fields beyond the input works as well, or is that too redundant?

Reply:
Yes, it should probably test that there are Gets done for the additional fields.